10/umbraco-deploy/.gitbook.yaml

```diff
@@ -4,3 +4,11 @@ root: ./
     readme: README.md
     summary: SUMMARY.md
 
+installing-deploy/installing-deploy: getting-started/install-configure-md
+installing-deploy/install-configure: getting-started-install-configure.md
+installing-deploy/install-configure/existing-site: getting-started/install-configure.md
+installing-deploy/install-configure/new-site: getting-started/install-configure.md
```

Comment:
You might be able to add the direct links here, if we want want. So, link to the specific expandable, which should also automatically open the expandable, I think 🤔 

Example: `getting-started/install-configure#new-site`